test/unittests/iotools/converters/oasis_to_bids/test_oasis_to_bids_utils.py

```diff
@@ -21,6 +21,8 @@ def clinical_data_path(tmp_path: Path) -> Path:
 def _build_clinical_data(clinical_data_path: Path) -> None:
     clinical_data_path.mkdir()
 
+    # todo :what happens if nan instead of value ? (handling of float...)
```

Comment:
Is it a note for yourself for this PR ?
If not, could you clarify it ?

Reply:
Totally for myself, I forgot to remove it